test/rubocop/cop/minitest/refute_instance_of_test.rb

```diff
@@ -66,26 +66,51 @@ def test_do_something
     RUBY
   end
 
-  def test_registers_offense_when_using_refute_with_instance_of_in_redundant_parentheses
+  def test_registers_offense_when_using_refute_equal_with_class_arguments
     assert_offense(<<~RUBY)
       class FooTest < Minitest::Test
         def test_do_something
-          refute((object.instance_of?(SomeClass)))
-          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer using `refute_instance_of(SomeClass, object)`.
```

Comment:
NOTE: Redundant parentheses will be resolved in https://github.com/rubocop/rubocop/pull/11739, so removed from this cop role.